Common/include/interpolation_structure.hpp

```diff
@@ -253,5 +285,98 @@ class CMirror : public CInterpolator {
    * \param[in] config - Definition of the particular problem.
    */
   void Set_TransferCoeff(CConfig **config);
+  
+};
+
+/*!
+ * \brief Sliding mesh approach
+  */
+class CSlidingMesh : public CInterpolator {
+public:
```

Comment:
I would leave that name since I'll probably need to re-use the very same class as a starting point for the implementation of the adaptive approach. 
So I'd keep it as general as possible for the time being, at least until the adaptive approach is ready.
We can still change the class name afterwards though.